xks-axum/Cargo.toml

```diff
@@ -9,9 +9,9 @@ publish = false
 
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 [dependencies]
-axum = { version = "0.5", features = ["headers"] }
+axum = { version = "0.6", features = ["headers"] }
```

Comment:
Why do we want to upgrade axum ?

Reply:
We want to proactively keep our software up-to-date with respect to the latest dependencies, instead of being forced to upgrade re-actively.